.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+### Jira Ticket:
+
+[XXX-YYY](https://choco.atlassian.net/browse/XXX-YYY)
+
+### Solution Description:
+
+<!-- [Short description] -->
+
+### Checklist:
+
+#### Tests
+- [ ] I tested my changes in local or tst environment using some of the Airflow DAGs. (See [README.md](/README.md))
+- [ ] I wrote the necessary unit tests for the changes I made.
+
+#### Downstream dependencies
+- [ ] Is it not Friday?
+- [ ] Is it not end of the Month?
```

Comment:
I don't think this make sense regarding dagger, because it's not deployed just because it's merged. Also this is a fork from a non Choco open source repository, so we shouldn't add choco related conecpts